packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-javascript-visitor.test.ts

```diff
@@ -109,22 +109,22 @@ describe('Javascript visitor', () => {
 
         export declare class SimpleNonModelType {
           readonly id: string;
-          readonly names?: (string | null)[];
+          readonly names?: (string | null)[] | null;
           constructor(init: ModelInit<SimpleNonModelType>);
         }
 
         export declare class SimpleModel {
           readonly id: string;
-          readonly name?: string;
-          readonly bar?: string;
+          readonly name?: string | null;
+          readonly bar?: string | null;
           readonly foo?: Bar[];
```

Comment:
Curious why optional model types can't be nullable as well

Reply:
Oh good catch. It should be `readonly foo?: Bar[] | null`